scripts/board.gd

```diff
@@ -30,6 +31,8 @@ func _ready():
 	self._draw_board()
 
 func get_piece_by_id(piece_id: String) -> Piece:
+	if not piece_id in self._pieces:
+		return null
 	return self._pieces[piece_id]
```

Comment:
get returns null if the key doesn't exist
```suggestion
	return self._pieces.get(piece_id)
```

Reply:
bda5a8d